types/ramda/test/modify-tests.ts

```diff
@@ -0,0 +1,16 @@
+import * as R from 'ramda';
+
+() => {
+    const person = { name: 'James', age: 20, pets: ['dog', 'cat'] };
+    R.modify('age', R.add(1), person); // => {name: 'James', age: 21, pets: ['dog', 'cat']}
+};
+
+() => {
+    const person = { name: 'James', age: 20, pets: ['dog', 'cat'] };
+    R.modify('pets', R.append('turtle'))(person); // => {name: 'James', age: 20, pets: ['dog', 'cat', 'turtle']}
+};
+
+() => {
+    const arr = [1, 2, 3];
+    R.modify(0, R.add(2), arr); // => [3, 2, 3]
+};
```

Comment:
I think there should be test cases for negative scenarios

Reply:
the first parameter indicates property of an object, not index of an array. so if I pass -1 as first parameter, the returned array would remain the same. should I write the case?